.github/workflows/build_workflow.yml

```diff
@@ -3,17 +3,27 @@ name: CI/CD Build Workflow
 on:
   push:
     branches: [master]
-    paths-ignore: ["docs/**"]
 
   pull_request:
     branches: [master]
-    paths-ignore: ["docs/**"]
```

Comment:
This line skips the required checks for docs-only changes, which doesn't allow merging PR